packages/x-date-pickers/src/MonthCalendar/PickersMonth.tsx

```diff
@@ -64,15 +64,15 @@ const PickersMonthButton = styled('button', {
   ],
 })<{
   ownerState: PickersMonthProps;
-}>(({ theme }) => ({
+}>(({ theme, ownerState }) => ({
   color: 'unset',
   backgroundColor: 'transparent',
   border: 0,
   outline: 0,
   ...theme.typography.subtitle1,
-  margin: '8px 0',
-  height: 36,
-  width: 72,
+  margin: `${ownerState.monthsPerRow === 3 ? 6 : 3}px 0`,
```

Comment:
I'm open to suggestions on this change.
it was not necessary.
I've only done it to be more in line with MD2 spacing spec if we are already adjusting it.